implementations/rust/ockam/ockam_api/src/authenticator/storage/authority_member.rs

```diff
@@ -0,0 +1,79 @@
+use ockam::identity::{Identifier, TimestampInSeconds};
+use ockam_core::compat::collections::BTreeMap;
+use ockam_core::compat::str::FromStr;
+use ockam_core::{Error, Result};
+
+/// Project member stored on the Authority node
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct AuthorityMember {
+    identifier: Identifier,
+    attributes: BTreeMap<Vec<u8>, Vec<u8>>,
+    added_by: Option<Identifier>,
+    added_at: TimestampInSeconds,
+    // Was provided by TrustedIdentities argument during the Authority startup
+    // pre-trusted identities can't be deleted using [`MembersStorage::delete_member()`]
+    is_pre_trusted: bool,
```

Comment:
Pre trusted is what it's used to call. Not saying it's perfect, but permanent also doesn't reflect how it's working - it's overwritten on every start of the Authority node with fresh values

Reply:
I don't think there is a right answer here. Modern sql dbs have a lot of things to support business logic, like stored functions and triggers. NoSQL seems to avoid that